template.yml

```diff
@@ -106,3 +227,9 @@ Outputs:
   BackendLambdaFunctionArn:
     Description: "ARN of the Backend Lambda"
     Value: !GetAtt BackendLambda.Arn
+  APIEndpoint:
+    Value: !Sub "https://${RestApi}.execute-api.${AWS::Region}.amazonaws.com/v1/um"
```

Comment:
You can remove `v1/um`

Reply:
done, should I also remove on line 189 for me the pathpattern: "v1/im" ?